community/modules/remote-desktop/chrome-remote-desktop/main.tf

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+locals {
+
+  user_startup_script_runners = var.startup_script == null ? [] : [
+    {
+      type        = "shell"
+      content     = var.startup_script
+      destination = "user_startup_script.sh"
+    }
+  ]
+
+  configure_nvidia_driver_runners = var.install_nvidia_driver == false ? [] : [
+    {
+      type        = "ansible-local"
+      content     = file("${path.module}/scripts/configure-grid-drivers.yml")
+      destination = "/usr/local/ghpc/configure-grid-drivers.yml"
+    }
+  ]
+
+  configure_chrome_remote_desktop_runners = var.configure_chrome_remote_desktop == false ? [] : [
+    {
+      type        = "ansible-local"
+      content     = file("${path.module}/scripts/configure-chrome-desktop.yml")
+      destination = "/usr/local/ghpc/configure-chrome-desktop.yml"
+    }
+  ]
+
+  driver     = { install-nvidia-driver = var.install_nvidia_driver }
+  logging    = var.enable_google_logging ? { google-logging-enable = 1 } : { google-logging-enable = 0 }
```

Comment:
I've actually removed this as the best practice is now to use [Ops agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent) instead of logging and monitoring agents.

There is an option to install ops agent through ansible [amongst others](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/fleet-installation) but I'm questioning how much value this would have for a remote desktop instance? 

I think we could add an optional parameter to add a ops agent ansible runner but maybe a todo later if this is worthwhile to do so

Reply:
sounds good.